index.js

```diff
@@ -17,14 +17,13 @@ module.exports = {
         'block-scoped-var': 2,
         'curly': 2,
         'dot-notation': 2,
-        'eqeqeq': 2,
+        'eqeqeq': [2, 'smart'],
```

Comment:
allows ` == null` to check for `null` and `undefined`, but not `false`, or other falsy values